presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/RowExpressionVerifier.java

```diff
@@ -354,6 +354,12 @@ private Boolean compareLiteral(Node expected, RowExpression actual)
     @Override
     protected Boolean visitStringLiteral(StringLiteral expected, RowExpression actual)
     {
+        if (actual instanceof CallExpression && functionResolution.isCastFunction(((CallExpression) actual).getFunctionHandle())) {
```

Comment:
This is needed for matching for example `cast('abc' as varchar)` with `expression` generated from string(since we now do not remove the cast in `RowExpressionTranslation`). If needed, I can move this to a separate commit.